Documentation/Diagnostics/PH2148.md

```diff
@@ -0,0 +1,69 @@
+# PH2148: Order DataRow attribute above TestMethod for unit tests
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MsTestAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MsTestAnalyzers) |
+| Diagnostic ID | PH2148 |
+| Category  | [MsTest](../MsTest.md) |
+| Analyzer | [DataRowOrderAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/main/Philips.CodeAnalysis.MsTestAnalyzers/DataRowOrderAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Warning |
+| Enabled By Default | Yes |
```

Comment:
enabled by default: No. (We'll enable it by default later after proving it's working as expected.)